server.js

```diff
@@ -20,8 +14,43 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+// http://localhost:8080/
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Top 50 popular songs on spotify");
+});
```

Comment:
in here you could return a documented list of your endpoints using [Express List Endpoints](https://www.npmjs.com/package/express-list-endpoints), which is also a basic requirement 👍